.github/workflows/build.yml

```diff
@@ -110,6 +135,28 @@ jobs:
           DEPLOY_NIGHTLY_GITHUB_TOKEN: ${{ secrets.DEPLOY_NIGHTLY_GITHUB_TOKEN }}
           DEPLOY_NIGHTLY_GITHUB_USER: leanprover-community-bot
 
+  tests:
+    name: run tests and docs
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      # number of commits to check for olean cache
+      GIT_HISTORY_DEPTH: 20
```

Comment:
```suggestion
```

Is this flag necessary in this job?

Reply:
No, that was only used for the "try to find olean cache" step.